examples/hosts/app-integration/external-data/src/mock-external-data-service/service.ts

```diff
@@ -48,10 +64,12 @@ export async function initializeExternalDataService(props: ServiceProps): Promis
 	/**
 	 * Mock webhook for notifying subscribers to changes in external data.
 	 */
-	const webhook = new MockWebhook<ITaskData>();
-
-	function notifyWebhookSubscribers(newData: ITaskData): void {
+	function notifyWebhookSubscribers(externalTaskListId: string, newData: ITaskData): void {
 		console.log(formatLogMessage("External data has changed. Notifying webhook subscribers."));
+		const webhook = webhookCollection.get(externalTaskListId);
+		if (webhook === undefined) {
+			throw new Error(`No webhook subscribers for ${externalTaskListId}`);
+		}
```

Comment:
I think here we can just `return; // No subscribers for this task list` instead of throwing. In the real world it could be a normal occurrence if some task list in the external data service changed and there were no subscriptions for it. I don't think the example app can get in that state but I don't think it serves much of a purpose for the example to throw here anyway.